usaspending_api/api_contracts/contracts/v2/autocomplete/awarding_agency.md

```diff
@@ -30,7 +30,6 @@ This route sends a request to the backend to retrieve awarding agencies matching
 + `id` (required, number)
 + `toptier_flag` (required, boolean)
 + `toptier_agency` (required, object)
-    + `id` (required, number)
```

Comment:
This was something I added last sprint in preparation for this ticket.  Turns out I didn't need it so I removed it.